### PR TITLE
[screensaver.picture.slideshow] 6.1.0

### PR DESCRIPTION
**Pull request by ronie**
### Description
code cleanup and update metadata parsers.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: screensaver.picture.slideshow/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+v6.1.0
```
Comment (enen92) [inline screensaver.picture.slideshow/changelog.txt:1]: drop if possible

File: screensaver.picture.slideshow/lib/exifread_xbmcvfs/utils.py [hunk 2]
```diff
@@ -40,23 +40,6 @@ def make_string_uc(seq):
     return make_string(seq)
```
Comment (enen92) [inline screensaver.picture.slideshow/lib/exifread_xbmcvfs/utils.py:40]: maybe move exifreadvfs to its own module?

File: screensaver.picture.slideshow/lib/iptcinfo3_xbmcvfs.py [hunk 3]
```diff
@@ -0,0 +1,900 @@
+#!/usr/bin/env python3
```
Comment (enen92) [inline screensaver.picture.slideshow/lib/iptcinfo3_xbmcvfs.py:1]: another bundled module

**Comment by ronie** [comment]
the two external modules have been modified by me to work with xbmcvfs.File() objects.
while i've verified the bits and pieces used by this addon work correctly, i can't guarantee every method provided by those modules still functions as they should.

as such, i'm a bit hesitant to submit them as separate modules to the community.

**Pull request merged**